extensions/gdata/module/langs/translation-en.json

```diff
@@ -44,6 +44,7 @@
 	"gdata-exporter": {
 		"uploading": "Uploading...",
 		"upload-error": "Upload error: ",
+  		"upload-success": "Project was exported successfully to google drive with Id ",
```

Comment:
@jackyq2015 Perhaps this should say "uploaded" instead of "exported" ?

Reply:
@jackyq2015 also capitalize "Google Drive" since it's a registered trademark.

Reply:
updated the json